wv2util/AppOverride.cs

```diff
@@ -777,6 +842,79 @@ public bool IsRuntimeEvergreenPreview
                 OnPropertyChanged("IsRuntimeEvergreenPreview");
             }
         }
+
+        public bool IsRuntimeStable
+        {
+            get => ReverseSearchOrder && RuntimePath == "" && ReleaseChannels.Contains("0");
+            set
+            {
+                if (value)
+                {
+                  ReverseSearchOrder = true;
+                  if (!ReleaseChannels.Contains("0"))

```

Comment:
If the release channels override isn't set then the default value is all of the channels correct? If you read an entry from the registry and there's no release channels value set then you need to interpret that as all checkboxes being checked. And vice versa if all check boxes become checked we should serialize to the registry as removing the release channels key.